gigascience/gigadb-website
# Sort datasets by upload status

# Pull request for issue: #2127

This is a pull request for the following functionalities:

* Make upload status column in admin dataset sortable and filterable

![image](https://github.com/user-attachments/assets/b81eafe8-30b5-462f-8db8-5b3d7fff3584)

## How to test?

Navigate http://gigadb.gigasciencejournal.com/adminDataset/admin and use the sorting and filtering functionality

Run docker-compose run --rm codecept run --no-redirect -g issue-2127 acceptance -vv and confirm passing tests

## How have functionalities been implemented?

Updated `CustomGridView` (CGridView) config so that filter and sorting are added for the uplaod status column

## Any issues with implementation?

--

## Any changes to automated tests?

Added tests to check for filtering and sorting for the column
